types/node-crate/index.d.ts

```diff
@@ -0,0 +1,65 @@
+// Type definitions for node-crate 2.0
+// Project: https://github.com/arobson/rabbot
+// Definitions by: Greg Jednaszewski <https://github.com/gjednaszewski>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+declare namespace crate {
+    interface DBResultObject {
+        json: [object];
```

Comment:
Do you intend these to be 1-tuples or arrays? As written they're 1-tuples, while `object[]` would be the array syntax.

Reply:
Thanks for the review; I made the change.